deploy-as-code/helm/environments/works-dev.yaml

```diff
@@ -205,6 +205,7 @@ project-management-system:
 muster-roll-service:
   tracing-enabled: true
   java-enable-debug: true
+  app-timezone: "Asia/Kolkata"

```

Comment:
Please use the already defined timezone key (https://github.com/egovernments/DIGIT-DevOps/blob/c43646aa0bfa0f770ac64025b0d0449c3542ff41/deploy-as-code/helm/environments/works-dev.yaml#L60).

Reply:
Removed this from works-dev.yaml and made necessary change in values.yaml